src/API/Site/Controllers/MerchantCenter/RequestReviewController.php

```diff
@@ -116,4 +124,28 @@ protected function get_schema_properties(): array {
 	protected function get_schema_title(): string {
 		return 'merchant_account_review';
 	}
+
+	/**
+	 * Save the Account Review Status data inside a transient for caching purposes.
+	 *
+	 * @param array $value The Account Review Status data to save in the transient
+	 */
+	private function set_cached_review_status( $value ): void {
+		$this->transient->set(
+			Transients::MC_ACCOUNT_REVIEW,
```

Comment:
In other classes we use the TransientsInterface for this (which means we can have one import less at the top of the file). We use the same pattern for options which means we are more flexible in case we ever use a different implementation to store options/transients.